Legacy/bonej/src/main/java/org/bonej/plugins/ParticleDisplay.java

```diff
@@ -466,7 +521,18 @@ private static boolean isRadiiValid(final double[] radii) {
 		return true;
 	}
 
+	/**
+	 * Limit value to within minimum and maxium values
+	 * 
+	 * @param value the input value
+	 * @param min the allowed minimum
+	 * @param max the allowed maximum
+	 * @return value, or min if value < min; max if value > max
+	 */
 	private static int clamp(int value, int min, int max) {
```

Comment:
I forgot to say in the last PR that you can document exceptions with
```
* @throws IllegalArgumentException if min >  max 
*/
private static int clamp(int value, int min, int max)  throws IllegalArgumentException {
```
Not only is it informative for developers, but your IDE will be clever enough to warn:
```
try {
  clamp(value, min, max);
} catch (Exception e) { // Should catch IllegalArgument here!
```